Python/Tests/Core/PathUtilsTests.cs

```diff
@@ -678,6 +685,31 @@ public class PathUtilsTests {
             AssertUtil.ContainsExactly(files.Select(f => Path.GetExtension(f).ToLowerInvariant()).ToSet(), ".exe");
         }
 
+        [TestMethod, Priority(0)]
+        public void TestFindFile() {
+            var root = TestData.GetPath("TestData");
+            // File is too deep - should not find
+            Assert.IsNull(PathUtils.FindFile(root, "9E90EF25FCE648B397D0CCEC67305A68.txt", depthLimit: 0));
```

Comment:
Do you need to add those 2 text files? I'm not finding them on disk, or in the PR.